libraries/AP_BLHeli/AP_BLHeli.cpp

```diff
@@ -1620,8 +1648,11 @@ void AP_BLHeli::update_telemetry(void)
     }
     if (now - last_telem_request_us >= telem_rate_us) {
         // ask the next ESC for telemetry
-        last_telem_esc = (last_telem_esc + 1) % num_motors;
-        uint16_t mask = 1U << motor_map[last_telem_esc];
+        uint16_t mask = 0;
+        do {
+            last_telem_esc = (last_telem_esc + 1) % num_motors;
```

Comment:
need to check that the (digital_mask & ((1<<num_motors)-1)) != 0 ?

Reply:
Fixed